viewer/packages/data-providers/src/types.ts

```diff
@@ -29,6 +29,7 @@ export type Image360Descriptor = {
 
 export type Image360Face = {
   face: 'front' | 'back' | 'left' | 'right' | 'top' | 'bottom';
+  mimeType: 'image/jpeg' | 'image/png';
```

Comment:
`jpeg` & `png` are the only format we support?

Reply:
For now, yes.